src/Business/Detection/Common/Repositories/IDetectionRepository.cs

```diff
@@ -2,9 +2,12 @@ namespace Business.Detection.Common.Repositories
 {
     using System.Collections.Generic;
     using Business.Detection.Common.Models;
+    using Business.Detection.Creating.Models;
 
     public interface IDetectionRepository
     {
         public IEnumerable<MDetection> All();
+
+        public MDetection Create(MCreateDetection item);
```

Comment:
Should we change this method to insert a List of MCreateDetection input and receive a List of MDetection? It should look like this: `public IEnumerable<MDetection> Insert(IEnumerable<MCreateDetection> input)`

Reply:
Let`s have a talk about this.

Reply:
Done.